doc/specs/#117-Windows-Store-Support.md

```diff
@@ -0,0 +1,144 @@
+---
+author: <kevin> <larkin> <kevinlams/<kevinla@microsoft.com>
+created on: <2020-05-28>
+last updated: <2020-05-28>
+issue id: <github issue id>
+---
+# ===== DRAFT ======
+# Package Manager Windows Store Support
+
+## Abstract
+
+
+This spec identifies requirements and implementation for supporting Windows Store apps in the Windows Package Manager.
+
+## Inspiration
+
+
+The Windows Store is a curated list of applications.  The store apps include tools and the best of class applications and must be available to Package Manager users.
+
+## Solution Design
+
+Windows Package Manager will consume a manifest file like it does with all other formats.  The Windows Package Manager will then use a store API to initiate downloads and install the apps.
+
+
+
+**Store Apps**
+The Store has multiple app types.  Those that are free for download and free, but with purchase options, as well as paid apps.
+
+***Free Apps***  
+Apps that are free, will be downloaded directly when requested.
+
+***Pay apps [NOT SUPPORTED]***
+Apps that are for purchase, will not be allowed to be submitted.  If an app from the store requires commerce, to run, we will not allow them in the Windows Package Manager repository.  
+These apps will fail to activate and therefore should not pass our approval policy.
+
+***In app purchase***
+Apps that require in app purchase, can be allowed, as long as the app can correctly handle activation and treat the in-app purchase out of band from the Windows Package Manager.
+
+## Manifest Requirements 
+To support store apps, the manifest file will need to be updated to support the new format.  Most fields remain the same, but additional changes are required to support the store.
+
+### YAML Spec updates
+<b>URL</b> the URL used for the store is not the same as the URL users and the developers expect.  Therefore this field is unnecessary.  
```

Comment:
I think we should retain the URL field and have it point to the store application https:// address

Reply:
Are you suggesting it would be required?  I think the URL you are suggesting would typically be the HomePage.